components/invitation_modal/result_table.test.tsx

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {shallow, mount} from 'enzyme';
+import {IntlProvider} from 'react-intl';
+
+import deepFreeze from 'mattermost-redux/utils/deep_freeze';
+
+import EmailIcon from 'components/widgets/icons/mail_icon';
+import AlertIcon from 'components/widgets/icons/alert_icon';
+import GuestBadge from 'components/widgets/badges/guest_badge';
+import BotBadge from 'components/widgets/badges/bot_badge';
+import Avatar from 'components/widgets/users/avatar';
+
+import ResultTable, {Props} from './result_table';
+
+let props: Props = {
+    sent: true,
+    rows: [],
+};
+type WithIntlProps = {
+    children: React.ReactNode | React.ReactNodeArray;
+}
+
+const WithIntl = (props: WithIntlProps) => {
```

Comment:
we already have a function called mountWithIntl for these scenarios `import {mountWithIntl} from 'tests/helpers/intl-test-helper';
`